modules/src/terminal.c

```diff
@@ -36,8 +37,18 @@ bool cursor = true;
 bool running = true;
 bool focused = true;
 
+color_scheme_t clr = {
+    .bg_color = 0x00353535,
+    .base_color = 0x00222221,
+    .highlight = 0x00030303,
+    .border_color = 0x00000000,
+    .text_color = 0x00FFFFFF,
+    .border_color2 = 0x00555555,
+};
+
 int main() {
     win = snow_open_window("Terminal", twidth, theight, WM_NORMAL);
+    win->color = &clr;
```

Comment:
Again, the color scheme here is only for the titlebar, hardcoding it in the snow_draw_window function makes more sense than extending the state of window objects with a one time use variable.